### PR TITLE
Added a page for polling to the commercial doc

### PR DESCRIPTION
**Pull request by malikmasis**
For the second option
closes https://github.com/volosoft/volo/issues/10073

### CODE REVIEW

File: en/modules/cms-kit/poll.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+You can list, create, update and delete on the admin side of your solution.
+
+![poll-page](../../images/cmskit-module-poll-page.png)
+![poll-edit-question-page](../../images/cmskit-module-poll-edit-question-page.png)
+![poll-edit-options-page](../../images/cmskit-module-poll-edit-options-page.png)
+
+## The Poll Widget
+
+The poll system provides a poll [widget](https://docs.abp.io/en/abp/latest/UI/AspNetCore/Widgets) users to vote and show the result.
+You can simply place the widget on a page like below:
```
Comment (ismcagdas) [inline en/modules/cms-kit/poll.md]:
```suggestion
The poll system provides a poll [widget](https://docs.abp.io/en/abp/latest/UI/AspNetCore/Widgets) for users to vote and show the result. You can place the widget on a page like the below:
```

**Review by ismcagdas — CHANGES_REQUESTED** [review]

**Review by ismcagdas — APPROVED** [review]

**Comment by ismcagdas** [comment]
@braim23 you can review now.

File: en/modules/cms-kit/index.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ And the following features are provided by the CMS Kit pro version:
 * [**Newsletter**](newsletter.md) system to allow users to subscribe to newsletters.
 * [**Contact form**](contact-form.md) system to allow users to write message to you.
 * [**URL forwarding**](url-forwarding.md) system to create URLs that redirect to other pages or external websites.
+* [**Poll**](poll.md) system to create polls on any resource, like a blog post or a page.
```
Comment (hikalkan) [inline en/modules/cms-kit/index.md]: Poll can be used in blog posts and pages in future version (currently now available, so should not be documented like that). However, it is only a use case. Polls are mainly independent components can be used anywhere in a website. My suggestion:

Poll system to create quick polls for users.

File: en/modules/cms-kit/poll.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## The Poll Widget
+
+The poll system provides a poll [widget](https://docs.abp.io/en/abp/latest/UI/AspNetCore/Widgets) for users to vote and show the result. You can place the widget on a page like the below:
+
+```csharp
+@await Component.InvokeAsync(
+    typeof(PollViewComponent),
+    new
+    {
+        widgetName = "poll"
```
Comment (hikalkan) [inline en/modules/cms-kit/poll.md]: A name like `my-poll-1` would be better to understand this is a custom name. Also, mention that you can concurrently have multiple active poll widgets in your application with different widget names.

File: en/modules/cms-kit/poll.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+```csharp
+Configure<CmsKitPollingOptions>(options =>
+    {
+        options.AddWidget("poll");
+    });
+```
+
+`CmsKitPollingOptions` properties:
+
+- `WidgetNames`: List of defined widgets in the poll system.
```
Comment (hikalkan) [inline en/modules/cms-kit/poll.md]:
```suggestion
- `WidgetNames`: List of defined widgets in the poll system. `options.AddWidget` method was a shortcut to add a new widget to this list.
```

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
Thanks @malikmasis good document. I made a few small improvement suggestions.

**Review by hikalkan — APPROVED** [review]

**Pull request merged**